@here/harp-datasource-protocol/lib/Theme.ts

```diff
@@ -386,11 +580,20 @@ export interface TextStyleDefinition {
 export interface GradientSkyParams {
     /** Sky type. */
     type: "gradient";
-    /** Color of the upper part of the gradient. */
+    /**
+     * @format color-hex
```

Comment:
Be consistent with the typedoc (look at the members below).

Reply:
Yay! It's a typo. Thanks